platform/mellanox/mlnx-sfpd/scripts/mlnx-sfpd

```diff
@@ -218,7 +218,7 @@ class MlnxSfpd:
 
                     sfp_state = sfp_value_status_dict.get(module_state, STATUS_UNKNOWN)
                     if sfp_state == STATUS_UNKNOWN:
-                        log_error("unknown module state {} on port {}".format(module_state, port))
+                        log_error("unknown module state {} on port(s) {}".format(module_state, port_list))
```

Comment:
what is the unknown status? it should be either in or out. why is this an error message?